includes/class-core.php

```
@@ -1490,6 +1482,13 @@ public function execute_scheduled_scan() {
 
             // Set HTML Content Type using headers parameter instead
             $headers = array( 'Content-Type: text/html; charset=UTF-8' );
+            
+            // FIX: Set "From" header to match Site Title and Admin Email
+            // This ensures Gmail sees the email as authenticated (if the server allows sending as this address)
+            $admin_email = get_option( 'admin_email' );
```

PHPMD (2.15.0)

Avoid unused local variables such as '$admin_email'. (undefined)

(UnusedLocalVariable)